### DISCUSSION TITLE
How to include jars that cannot be included via pom.xml

### DISCUSSION THREAD

**Discussion by Exnadella** [buildpacks/community#162] [Q&A] [answered]
Hello Team,

We are including a commercial version of .jar package (Transaction manager) via classpath.  We cannot build a build packer for this commercial product.  How to include these using the buildpacks?

We can use the maven plugin to read the file from a src location, how will the buildpack detect and add the dependency to the image?  
How can we integrate this via CI/CD using GithubActions?

Do buildpacks offer any specific plugin to include the mandatory dependencies from a remote source location?

Thanks,
Ranganath.

**Comment by jromero** [comment, marked as answer]
There are multiple buildpack providers that may or may not support the functionality you desire. Here's a list of suggested buildpack providers (alphabetical):

- [Google buildpacks](https://github.com/GoogleCloudPlatform/buildpacks)
- [Heroku buildpacks](https://elements.heroku.com/buildpacks)
- [Paketo buildpacks](https://paketo.io/)

If need be, you could always [create your own buildpack](https://buildpacks.io/docs/buildpack-author-guide/create-buildpack/).

**Reply by Exnadella** [reply]
Hi Jromero,

We are using Paketo buildpacks, how to include custom Artifacts into container images which are built using Paketo Buildpacks via Spring Boot maven plugin with a build-image goal.
